src/assets/dbObjects/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Where is this file used?

Reply:
No where, I forgot that json file made it in.